docs/source/markdown/podman-volume-inspect.1.md

```diff
@@ -33,10 +54,41 @@ Print usage statement
 
 ```
 $ podman volume inspect myvol
-
+[
+     {
+          "Name": "myvol",
+          "Driver": "local",
+          "Mountpoint": "/home/dwalsh/.local/share/containers/storage/volumes/myvol/_data",
```

Comment:
do you want to include `/home/dwalsh` here?  I'd suggest something like `/home/myusername`